ts/testUtils.ts

```diff
@@ -56,6 +63,36 @@ export const assertStderr = (output: Output, expected: string) => {
   }
 };
 
+/*
+ * Build and retrieve the store path of specific attribute of a given flake.
+ * This is useful for testing, as it allows to check the output of a specific
+ * derivation.
+ */
+export const getStorePath = (flakeDir: string, attribute: string): string => {
```

Comment:
I have a very similar function called `buildPackage` in #424 (in `testUtils.ts`). Overall the API of `buildPackage` is closer to how a `garn` user would interact with something of type `Package`. So `buildPackage` is what a garn user could do by running `garn build ...` in a terminal. So maybe that would make it easier to write tests that are closer to real use-cases. (Also `buildPackage` renders the flake file for you, so that's convenient.)

Do you think `buildPackage` would work for your test?

(There's some things that aren't ideal about `buildPackage`, for example I didn't dry up `nixpkgs` which you did and is a good idea. But to me it seems that `getStorePath` and `buildPackage` should be the same function.)

Reply:
We're merging #424 now. I hope that makes it easier to figure out whether `buildPackage` works for you.

Reply:
I removed getStorePath and refactored the test to use buildPackage instead. I had to make slight modifications to some testUtils in order to get `system` into scope which is required by the expression generated by mkNpmProject.